bevy_ios_safearea/src/plugin.rs

```diff
@@ -12,8 +17,9 @@ use bevy::{ecs::system::SystemParam, prelude::*};
 /// fn bevy_system(safe_area: IosSafeArea) {
 ///     let safe_area_top = safe_area.top();
 /// }
-// ```
-#[derive(Resource, Clone, Debug, Default)]
+/// ```
+#[derive(Resource, Clone, Debug, Default, Reflect)]
+#[reflect(Resource)]
 pub struct IosSafeAreaResource {
```

Comment:
IosSafeAreaResource should be renamed `SafeAreaResource` then ; and crate renamed ?

Reply:
@ThierryBerger I dont think crates.io allows a rename, no?

Reply:
yeah, probably spawn a new crate then..?

Another option is to create a "parent" crate, but I'm not sure maintenance is worth it (or include them in the same repository and it may be ok?)
- window_safe_area
  - ios_safe_area
  - android_safe_area
  - windows_safe_area -> yeah naming is hard ok

(I also think such crate would be more useful if it could be not tied to bevy, that may be another subject, but if all we need is a window handle it makes sense, that would mean a new crate `bevy_window_safe_area` or an exposed feature...)

Reply:
@extrawurst I think one package is more than enough for it, but I leave it up to you

Reply:
the dependency hell alone makes me want these in two separate crates

Reply:
we could argue to start with `bevy_android_safearea` before doing the combined one right away